quickstack/manifests/compute_common.pp

```diff
@@ -409,11 +409,27 @@
        "puppet:///modules/sensu/plugins/disk-metrics.rb"
     ]
   }
-
+  
   class {'quickstack::ntp':
     servers => $ntp_local_servers,
   }
 
+  class { 'filebeat':
+    outputs => {
+      'logstash'  => {
+        'hosts'       =>  [$elasticsearch_host],
+        'loadbalance' => true
+      }
+    },
+    logging => {
+      'level' => "info"
+    }
+  }
+
+  filebeat::prospector { 'generic':
+      paths => ["/var/log/*.log", "/var/log/secure", "/var/log/messages", "/var/log/ceph/*", "/var/log/nova/*", "/var/log/neutron/*", "/var/log/openvswitch/*", "/var/log/cinder/", "/var/log/glance/", "/var/log/horizon/", "/var/log/httpd/", "/var/log/keystone/*"]
+  }
```

Comment:
I guess it should be /var/log/cinder/* and same with other folders. I added this in previous comment, but github removed the trailing *. You can fix this.